development-defaults.ini

```diff
@@ -165,9 +165,7 @@ stripe_error = "Stripe Error Override"
 
 [[door_payment_method]]
 cash   = "Pay with cash"
-stripe = "Pay with credit card now (faster)"
-manual = "Pay with credit card at the registration desk (slower)"
-group  = "Taking an unassigned Group badge (group leader must be present)"
+manual = "Pay with credit card at the registration desk"
```

Comment:
In that case, I'll have to reverse the changes in config.py to remove these options instead of adding them.